roles/pulp_redis/README.md

```diff
@@ -3,6 +3,11 @@ pulp_redis
 
 Install and start Redis, and install RQ in the Pulp virtualenv.
 
+Role Variables
+--------------
+
+* `pulp_redis_bind`: Interface and Port where Redis service will listen. One can specify a unix path (ie. 'unix:/var/run/redis/redis.sock'). Defaults to `127.0.0.1:6379`
```

Comment:
Can this be line wrapped to 100 chars. We need to add a check that should tell you instead of me (sorry about this). We strive to have all docs line wrap to 100 chars.